File: Proyectos/Python/Divisas.py
from forex_python.converter import CurrencyRates

# Crear una instancia del conversor de divisas
c = CurrencyRates()

# Definir las monedas y la cantidad que deseas convertir
amount = 100  # Cambia esto a la cantidad que quieras convertir
from_currency = "USD"
to_currency = "EUR"

# Realizar la conversión
result = c.convert(from_currency, to_currency, amount)

print(f"{amount} {from_currency} es igual a {result} {to_currency}")

File: Html/js/Divisas.js
document.getElementById("convert").addEventListener("click", function () {
    const amount = parseFloat(document.getElementById("amount").value);
    const fromCurrency = document.getElementById("fromCurrency").value;
    const toCurrency = document.getElementById("toCurrency").value;
    const exchangeRates = {
        USD: 1,
        EUR: 0.85,
    };

    const result = (amount * exchangeRates[toCurrency]) / 
    exchangeRates[fromCurrency];

    document.getElementById("result").innerHTML = `
    ${amount} ${fromCurrency} = ${result.toFixed(2)} ${toCurrency}`;
});
